packages/finance/src/liability/mod.rs

```diff
@@ -15,9 +15,14 @@ use crate::{
 };
 
 pub use self::level::Level;
+pub use self::liquidation::check_liability;
```

Comment:
suggest introducing this function as `check` only to have `liquidation::check(...)` at the calling sites